extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -378,18 +362,45 @@ export class AmpIframe extends AMP.BaseElement {
   /**
    * Updates the element's dimensions to accommodate the iframe's
    *    requested dimensions.
-   * @param {number|undefined} newWidth
-   * @param {number|undefined} newHeight
+   * @param {number|undefined} height
+   * @param {number|undefined} width
    * @private
    */
-  updateSize_(newHeight, newWidth) {
+  updateSize_(height, width) {
     if (!this.isResizable_) {
       user.warn(TAG_,
           'ignoring embed-size request because this iframe is not resizable',
           this.element);
       return;
     }
-    this.attemptChangeSize(newHeight, newWidth);
+
+    let newHeight, newWidth;
+    if (height !== undefined) {
+      newHeight = Math.max(
+        (this.element./*OK*/offsetHeight - this.iframe_./*OK*/offsetHeight)
+            + height,
+        height);
+    }
+    if (width !== undefined) {
+      newWidth = Math.max(
+        (this.element./*OK*/offsetWidth - this.iframe_./*OK*/offsetWidth)
+            + width,
+        width);
+    }
+
+    if (newHeight !== undefined || newWidth !== undefined) {
+      this.attemptChangeSize(newHeight, newWidth, () => {
+        this.iframe_.height = height;
```

Comment:
i also think we should not be setting the iframe height anymore since change size does exactly that